src/config/preferences.c

```diff
@@ -2276,7 +2276,7 @@ _get_default_string(preference_t pref)
     case PREF_STATUSBAR_ROOM_TITLE:
         return "name";
     case PREF_STATUSBAR_TABMODE:
-        return "default";
+        return "static";
```

Comment:
users will have `default` in their config file I guess? So it won't find that setting. That means we need to add migration (there is a part where we do this for other settings already). However for simplicities sake I would just continue to use `default` for static. (So this also means to make the changes above). Alternative is to add migration.

Reply:
> users will have `default` in their config file I guess? So it won't find that setting. That means we need to add migration (there is a part where we do this for other settings already). However for simplicities sake I would just continue to use `default` for static. (So this also means to make the changes above). Alternative is to add migration.

in code logic, this doesn't matter since only "dynamic" and "actlist" are checked.